Makefile

```diff
@@ -0,0 +1,45 @@
+# Variables
+DCURL_DIR=dcurl
+ROCKSDB_DIR=rocksdb
+ROCKSDB_BRANCH=5.18.fb
+ROCKSDB_VERSION=5.18.1
+
+.PHONY: all dcurl-build rocksdb-build iri-build check clean
+
+# Rules
+all: dcurl-build rocksdb-build iri-build
+
+${DCURL_DIR}:
+	git clone https://github.com/DLTcollab/dcurl.git $@
+
+${ROCKSDB_DIR}:
+	git clone https://github.com/DLTcollab/rocksdb.git $@
```

Comment:
How about replacing `git clone` with `git submodule` operations? Thus, we can avoid changing URI in Makefile.

Reply:
Done.